app/jobs/sync_custom_filter_count_job.rb

```diff
@@ -0,0 +1,9 @@
+class SyncCustomFilterCountJob < ApplicationJob
+  queue_as :low
+
+  def perform(filter)
+    return if filter.fetch_record_count_from_redis.nil?
```

Comment:
What if this data is stale? We don't set the data with expiry right?

Reply:
No. We will update the data only when a new conversation is created, considering that will be the only trigger to update the filter records count.

But if we can have a job to update the count too, after TTL.